insightfinder/README.md

```diff
@@ -4,24 +4,21 @@
 
 [InsightFinder][1] AIOps identifies system and application issues before they impact users. Powered by unsupervised machine learning, InsightFinder continuously learns from events, logs, metrics, and changes to detect anomalies, predict incidents, and remediate outages.
 
-InsightFinder ingests data from Datadog through standard APIs and finds anomalous events before your business is impacted.
+This bidirectional integration provides advanced AIOps capabilities. InsightFinder ingests data from Datadog through standard APIs and finds anomalous events before your business is impacted. Alerts for these anomalous events can then be sent into Datadog to notify your team.
 
 ## Setup
 
 ### Installation
 
-To configure the integration and send data to InsightFinder, see the [InsightFinder-Datadog Integration][2]. You need a Datadog [API key and application key][5].
+To configure the integration and send data to InsightFinder, see the [InsightFinder-Datadog Integration][2]. You need a Datadog [API key and application key][3].
 
-If you do not have an InsightFinder account, [sign up][3] to start your free trial.
 
 ## Support
 
-Contact [Datadog support][6] or email [InsightFinder support][4].
+Contact [Datadog support][4] or email [InsightFinder support][1].
```

Comment:
I think the support link is wrong? It links to the InsightFinder homepage.

Reply:
done